internal/locate/region_request.go

```diff
@@ -426,17 +434,17 @@ func (state *tryFollower) next(bo *retry.Backoffer, selector *replicaSelector) (
 	if err != nil || rpcCtx == nil {
 		return rpcCtx, err
 	}
-	if state.fallbackFromLeader {
-		staleRead := false
-		rpcCtx.contextPatcher.staleRead = &staleRead
+	if !state.fromAccessKnownLeader {
 		replicaRead := true
 		rpcCtx.contextPatcher.replicaRead = &replicaRead
 	}
+	staleRead := false
+	rpcCtx.contextPatcher.staleRead = &staleRead
```

Comment:
```suggestion
	if !state.fromAccessKnownLeader {
		staleRead := false
	        rpcCtx.contextPatcher.staleRead = &staleRead
	}
	replicaRead := true
	rpcCtx.contextPatcher.replicaRead = &replicaRead
```

Should it be like this?

Reply:
No, if it's from `accessKnownLeader`, replica read should not be set.

Reply:
The logic is consistent with master branch. https://github.com/tikv/client-go/blob/master/internal/locate/region_request.go#L461